databases/week2/hw2-1.sql

```diff
@@ -0,0 +1,70 @@
+--Part1
+use hyf;
+--ADD new task
+INSERT INTO task (id,title,description,created,updated,due_date,status_id,user_id)
+VALUES (16,'Finish Homework', 'Complete all SQL tasks for the week', NOW(), NOW(), '2025-02-15', 1, 2);
+
+
+
+--Change the title of task
+UPDATE task set title='submit Homework' WHERE id=1;
+
+--Change a task due date 
+UPDATE task SET due_date='2025-02-20' WHERE id=1;
+
+--Change a task status
+UPDATE task SET status_id=2 WHERE id=1;
+
+--Mark a task as complete
+UPDATE task SET description='COMPLETE' WHERE id=5;
+
+--Delete task 
+DELETE
+FROM task
+WHERE title='wash the car';
+
+----Part 3
+
+--1-Get all the tasks assigned to users whose email ends in @spotify.com
+SELECT * FROM task t
+LEFT JOIN user u
+on u.id=t.user_id
+where u.email like '%@spotify.com';
```

Comment:
As you are using the wrong data, this query is too simple. If you update the code to use the data for week 2, you will have to use two joins.